modules/n1ql/pages/n1ql-language-reference/with-recursive.adoc

```diff
@@ -251,108 +614,67 @@ WITH RECURSIVE similar_items AS (
 CYCLE id RESTRICT
 SELECT * FROM similar_items;
 ---- 
-====
-
-.Example finding all possible routes leaving from the TLV airport and going to 2 other airports
-====
-[source,sqlpp]
-----
-CREATE INDEX srcdestidx ON `travel-sample`.inventory.route( sourceairport, destinationairport );
-
-WITH RECURSIVE recroute AS (
-    SELECT sourceairport, destinationairport, 0 as depth,
-    [ sourceairport, destinationairport ] as route
-    FROM `travel-sample`.inventory.route 
-    WHERE sourceairport="TLV"
-        UNION
-    SELECT r.sourceairport, r.destinationairport, 
-    ARRAY_APPEND ( recroute.route, r.destinationairport ) as route,
-        recroute.depth+1 as depth
-    FROM `travel-sample`.inventory.route r JOIN recroute ON 
-        recroute.destinationairport = r.sourceairport
-    WHERE r.sourceairport!="TLV" and recroute.depth<2 
-)
-SELECT * FROM recroute LIMIT 2000;
-----
 
 .Results
```

Comment:
No need to repeat the results here: you could just have a line saying "The results are the same as for the previous query."